Littlefs/README.md

```diff
@@ -0,0 +1,126 @@
+# Littlefs project
```

Comment:
This name is identical to the name of the underlying open source project that it uses -- use a different name to make it clear that this builds on, but is not a fork or copy of, that project